buildpack/runtime_components/storage.py

```diff
@@ -53,6 +60,9 @@ def _get_s3_specific_config(vcap_services, m2ee):
 
     access_key = os.getenv("S3_ACCESS_KEY_ID", access_key)
     secret = os.getenv("S3_SECRET_ACCESS_KEY", secret)
+    tvm_endpoint = os.getenv("S3_TVM_ENDPOINT", tvm_endpoint)
+    tvm_username = os.getenv("S3_TVM_USERNAME", tvm_username)
+    tvm_password = os.getenv("S3_TVM_PASSWORD", tvm_password)
```

Comment:
I think it's good to document these environment variables here as well: https://github.com/mendix/cf-mendix-buildpack#s3-settings